libraries/ESP8266WiFi/src/ESP8266WiFiType.h

```diff
@@ -156,5 +156,6 @@ struct WiFiEventSoftAPModeProbeRequestReceived
     uint8 mac[6];
 };
 
+typedef void (*sc_pass_decrypt_fn_t)(char* decrypted, const char* encString);

```

Comment:
A matter of preference, I'm just suggesting: perhaps use `Cb` instead of `Fn` and stick to C++ camelCase for the function type as well? Could even use `std::function`.